nestkernel/event.h

```diff
@@ -178,7 +178,7 @@ class Event
    *
    * @see NEST Time Memo, Rule 3
    */
-  long get_rel_delivery_steps( const Time& t ) const;
+  long get_rel_delivery_steps( const Time& t );
```

Comment:
Could you add a note in the doxygen comment for the method why it is not `const`? It will make it easier to understand the code, given that developers usually assume that getters are const.

Alternatively, one could keep the method `const` and declare `stamp_steps_` a `mutable`. @apeyser, do you have an opinion on which approach would make for more maintainable code in the long run?